app/screens/send-bitcoin-screen/send-bitcoin-success-screen.tsx

```diff
@@ -46,11 +48,14 @@ const SendBitcoinSuccessScreen = ({
       contentContainerStyle={styles.contentContainer}
     >
       <View style={styles.Container}>
-        <GaloyIcon name={"payment-success"} size={128} />
-
-        <Text {...testProps("Success Text")} style={styles.successText}>
-          {LL.SendBitcoinScreen.success()}
-        </Text>
+        <Animated.View entering={PinwheelIn.duration(3000).springify().delay(50)}>
```

Comment:
what is the `3000` and `50` for? those magic value should be set as const to be documented.

Reply:
I've refactored it into a component to keep the config in one place.